Filter/ClassFilter.php

```diff
@@ -13,10 +13,11 @@
 
 use Sonata\AdminBundle\Datagrid\ProxyQueryInterface;
 use Sonata\CoreBundle\Form\Type\EqualType;
-use Symfony\Component\Form\Extension\Core\ChoiceList\ChoiceList;
 
 class ClassFilter extends Filter
 {
+    const OPTION_SUB_CLASSES = 'sub_classes';
```

Comment:
Is there any benefit to making this publicly accessible (via `ClassFilter::OPTION_SUB_CLASSES`)?
Do we usually expose options with a constant?